packages/go_router_builder/lib/src/route_config.dart

```diff
@@ -105,6 +120,39 @@ class RouteConfig {
   final String _path;
   final InterfaceElement _routeDataClass;
   final RouteConfig? _parent;
+  final String? _key;
+  final bool _isShellRoute;
+
+  static String? _decodeKey(InterfaceElement classElement) {
```

Comment:
```suggestion
  static String? _generateNavigatorKeyGetterCode(InterfaceElement classElement) {
```

Reply:
Done in https://github.com/flutter/packages/pull/3439/commits/61c3d9d077012650e0ce82ffb51f25c0892bac20